npep/npep-137-conformance-profiles.md

```diff
@@ -0,0 +1,187 @@
+# NPEP-137: Conformance Profiles
+
+* NOTE: Adapted with love from https://gateway-api.sigs.k8s.io/geps/gep-1709/ (Then
```

Comment:
NIT add this blurb in the TLDR section not here.

Reply:
ack moving this..